conf.d/metrics.conf

```diff
@@ -68,6 +68,23 @@ stdout_logfile_maxbytes=0
 stderr_logfile=/dev/stderr
 stderr_logfile_maxbytes=0
 
+[program:graphite_exporter]
+command=twistd --nodaemon
+    --pidfile=%(program_name)s.pid vumi_worker
+    --hostname %(ENV_AMQP_HOST)s
+    --port %(ENV_AMQP_PORT)s
+    --vhost %(ENV_AMQP_VHOST)s
+    --username %(ENV_AMQP_USERNAME)s
+    --password %(ENV_AMQP_PASSWORD)s
+    --set-option=metrics_host:localhost
+    --set-option=metrics_port:9109
```

Comment:
Instead of this whole block (from `command=twistd ...` to the last `--set-option=...`), we want `command=/graphite_exporter` here.